README.md

```diff
@@ -10,19 +10,24 @@ and [BOSH][] manifests that allow provisioning of [CloudFoundry][] on AWS.
 [BOSH]: https://bosh.io/
 [CloudFoundry]: https://www.cloudfoundry.org/
 
-## Glossary
+## Overview
 
-- Environment: a single Cloud Foundry installation and its supporting
-  infrastructure.
-- Bootstrap Concourse: is responsible for creating or destroying a *Deployer
-  Concourse* to an *environment*. You don't need to keep this once you
-  have a *Deployer Concourse* running.
-- Deployer Concourse: is responsible for deploying everything else within
-  its *environment*, e.g. BOSH and CloudFoundry. It should be kept running
-  while that *environment* exists.
+The following components needs to be deployed in order. They should be
+destroyed in reverse order so as not to leave any orphaned resources:
+
+1. [Bootstrap Concourse](#bootstrap-concourse)
+1. [Deployer Concourse](#deployer-concourse)
+1. [MicroBOSH](#microbosh)
+1. [CloudFoundry](#cloudfoundry)
 
 ## Bootstrap Concourse
 
+This runs outside an environment and is responsible for creating or
```

Comment:
I think we should keep the definition of an environment. Inline here or in a specific section above.